api/metrics/annually.py

```diff
@@ -0,0 +1,29 @@
+from api.utils.database import rows_to_dicts
+
+
+class AnnuallyMetrics:
+    """
+    Metrics for annually timeline view.
+    """
+
+    def __init__(self, con):
+        self.con = con
+
+    def belonging(self, segment):
+        """
+        Returns the rate of belonging of a segment per year
+        """
+        query = """
+        SELECT
+            period_end_year || "-01-01" as week,
+            value / 100 AS value,
+            segment
+        FROM belonging
+        WHERE layer = "place"
+        AND segment == "{segment}"
```

Comment:
Because this is wrapped in quotes, we don't have a risk of SQL injection (e.g., if someone asked for segment = ";drop tables;"). So this should be safe.